### ISSUE TITLE
(Feature Request)

### ISSUE THREAD

**Issue by ChrisColotti** [jakewaldron/PlexEmail#79]
Enable SSL on web along with specifying a port.  Since most users are hosting PLEX in their home it's not the best nor maybe port 80 is not available to open.  Allowing SSL as well as a port designation would allow a little more security to use/access the web portion.  Or hosting on a separate server (see my other posting) where you can isolate the app/web portion.

**Comment by RobertSmits** [comment]
As far as I know you need to run the web sever yourself.  So the port and SSL support are entirely web server dependent.
Personally I host my PlexEmail site using Apache and HTTPS so I dont really see why you can't do the same,
just make sure you set the correct url in your config. 
As an example, this is my web section:

```
##Web
web_enabled = True
#This will not create the index.html file, but will still save the images to the images folder.  (Useful for when you only want to send an email)
web_only_save_images = False
web_domain = 'https://plexmail.example.com'
web_path = ''
web_delete_previous_images = True
web_skip_if_no_additions = False
```

**Comment by ChrisColotti** [comment]
Yeah that's the way I set it up.  It would just be nice if somehow it could be truly de-coupled from the PMS.  The docker containers out there I could map the PMS folder but the didn't build them with decent python so the scripts failed.  I don't know how to build a container image so maybe I'll try :).
